Hack to fix ppu-strip in SDK 3.0 toolchain.

Introduced in linux-2.6.git/366234f657879aeb7a1e2ca582f2f24f3fae9269,
Update linker script to properly set physical addresse
Author: Kumar Gala <galak@kernel.crashing.org>  2008-04-15 12:52:28

BFD: /target/tool/boot/vmlinux.strip: section .text lma 0x0 overlaps previous sections
BFD: /target/tool/boot/vmlinux.strip: section .ref.text lma 0x335000 overlaps previous sections
BFD: /target/tool/boot/vmlinux.strip: section .devinit.text lma 0x335eec overlaps previous sections
BFD: /target/tool/boot/vmlinux.strip: section .meminit.text lma 0x337454 overlaps previous sections
BFD: /target/tool/boot/vmlinux.strip: section .rodata lma 0x339000 overlaps previous sections


---
 arch/powerpc/kernel/vmlinux.lds.S |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/powerpc/kernel/vmlinux.lds.S
+++ b/arch/powerpc/kernel/vmlinux.lds.S
@@ -7,6 +7,8 @@
 #include <asm-generic/vmlinux.lds.h>
 #include <asm/cache.h>
 
+#define AT(_x)
+
 ENTRY(_stext)
 
 #ifdef CONFIG_PPC64
